build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+python src/etc/build_rust.py $@
```

Comment:
I think the only python2 feature that was used in `build_rust.py` and `build_llvm.py` is the `print` statement. I have changed them to `print()` which is available in both python2 and python3. This should make the scripts work with either version.